src/bin/wc.rs

```diff
@@ -104,23 +104,48 @@ struct Flags {
 }
 
 impl Flags {
-    fn print_count<'a, W: Write>(self, count: Counter, file: &'a str, stdout: &mut W, stderr: &mut Stderr) {
-        stdout.write(b"    ").try(&mut *stderr);
+    fn print_counts<W: Write>(self, mut counts: Vec<(Counter, String)>, stdout: &mut W, stderr: &mut Stderr) {
```

Comment:
You're copying 3 `usize` amounts of data, rather than copying the 1 `usize` from borrowing its address instead.